include/deal.II/lac/scalapack.h

```diff
@@ -180,21 +180,106 @@ class ScaLAPACKMatrix : protected TransposeTable<NumberType>
    */
   void invert();
 
+
```

Comment:
sorry to be PITA, in headers we keep 1 empty line. Only in .cc we keep 3 so that it's easier to read. Please adjust the header.